### PR TITLE
Don't report load errors to server.

### LINKED ISSUE #5726: Stop reporting 404s via error reporting
It is just spammy and not useful.

### PR DESCRIPTION
**Pull request by cramforce**
Fixes #5726

### CODE REVIEW

File: src/event-helper.js [hunk 1]
```diff
@@ -17,6 +17,9 @@
 import {timerFor} from './timer';
 import {user} from './log';
 
+/** @const {string}  */
+const LOAD_FAILURE_PREFIX = 'Failed to load:';
```
Comment (erwinmombay) [inline src/event-helper.js]: oh nevermind, didnt realize this is actually what we use

File: src/event-helper.js [hunk 2]
```diff
@@ -138,7 +141,16 @@ function failedToLoad(event) {
...
+  throw user().createError(LOAD_FAILURE_PREFIX, target);
+}
+
+/**
+ * Returns true if this error message is was created for a load error.
+ * @param {string} message An error message
+ * @return {boolean}
+ */
+export function isLoadErrorMessage(message) {
+  return message.indexOf(LOAD_FAILURE_PREFIX) != -1;
```
Comment (jridgewell) [inline src/event-helper.js]: `#startsWith`?

Comment (cramforce) [inline src/event-helper.js]: These errors sometimes get mangled. We want to catch them all.

File: src/event-helper.js (hunk 1 above)
Comment (cramforce) [inline src/event-helper.js]: But I use the constant in both places.

**Pull request merged**